.github/workflows/build_and_test.yml

```diff
@@ -111,13 +111,13 @@ jobs:
         key: ${{ matrix.java }}-${{ matrix.hadoop }}-maven-${{ hashFiles('**/pom.xml') }}
         restore-keys: |
           ${{ matrix.java }}-${{ matrix.hadoop }}-maven-
-    - name: Cache Ivy local repository
+    - name: Cache Coursier local repository
       uses: actions/cache@v2
       with:
-        path: ~/.ivy2/cache
-        key: ${{ matrix.java }}-${{ matrix.hadoop }}-ivy-${{ hashFiles('**/pom.xml', '**/plugins.sbt') }}
+        path: ~/.cache/coursier
```

Comment:
Thank you, @sarutak . BTW, I cannot find this directory in my Mac. Is it dependent on OS?
```
$ ls ~/.cache/coursier
ls: /Users/dongjoon/.cache/coursier: No such file or directory
```

Reply:
The directories are different in OSs (https://get-coursier.io/docs/cache)

Reply:
Thanks! @HyukjinKwon .

Reply:
> BTW, I cannot find this directory in my Mac. Is it dependent on OS?

Yeah, seems so.
On macOS, it is `~/Library/Caches/Coursier`.